upgrades/preupgradesteps.go

```diff
@@ -17,11 +17,11 @@ import (
 // PreUpgradeStepsFunc is the function type of PreUpgradeSteps. This may be
 // used to provide an alternative to PreUpgradeSteps to the upgrade steps
 // worker.
-type PreUpgradeStepsFunc func(_ *state.StatePool, _ agent.Config, isController, isMaster, isCaas bool) error
+type PreUpgradeStepsFunc = func(_ *state.StatePool, _ agent.Config, isController, isCaas bool) error
```

Comment:
Like changing the name, suggest isPrimaryController instead.  We'd only run the db steps on the controller node, which is primary db, during upgrade.  See also worker.go:123

Reply:
`isMaster` is gone. `isController` is correctly named in that it differentiates from other machine/unit/model agents.